### ISSUE TITLE
Files missing error - change location to store extracted TagUI files [fixed]

### ISSUE THREAD

**Issue by kensoh** [tebelorg/TagUI-Python#3]
The default unzip location is tagui subfolder under user's temporary folder.

However, after some days, operating system may delete files partially within the tagui folder, making it inaccurate to assume that if tagui executable file exists, the installation is working.

Raising issue to iterate on more persistent locations to place the unzip tagui folder. For eg ~/.tagui on Linux and macOS, %APPDATA%\tagui on Windows.

**Comment by kensoh** [comment]
Prior to next update, workaround is calling t.setup() if some unknown file missing message is shown.

**Comment by kensoh** [comment]
Fixed with v1.1, to apply this fix to avoid TagUI files in temp folder being deleted by OS -
```
pip install tagui --upgrade
```

**Comment by kensoh** [comment]
Fixed with v1.1, to apply this fix to avoid TagUI files in temp folder being deleted by OS -
```
pip install tagui --upgrade
```

**Comment by kensoh** [comment]
Fixed with v1.1, to apply this fix and avoid TagUI files in temp folder being deleted by OS -
```
pip install tagui --upgrade
```
